Mathlib/Analysis/InnerProductSpace/LinearPMap.lean

```diff
@@ -243,3 +243,32 @@ theorem toPMap_adjoint_eq_adjoint_toPMap_of_dense (hp : Dense (p : Set E)) :
 #align continuous_linear_map.to_pmap_adjoint_eq_adjoint_to_pmap_of_dense ContinuousLinearMap.toPMap_adjoint_eq_adjoint_toPMap_of_dense
 
 end ContinuousLinearMap
+
+section Star
+
+namespace LinearPMap
+
+variable [CompleteSpace E]
+
+instance instStar : Star (E →ₗ.[𝕜] E) where
+  star := fun A ↦ A.adjoint
+
+variable {A : E →ₗ.[𝕜] E}
+
+theorem isSelfAdjoint_def : IsSelfAdjoint A ↔ A† = A := by
+  rfl
+
+theorem _root_.IsSelfAdjoint.dense_domain (hA : IsSelfAdjoint A) : Dense (A.domain : Set E) := by
```

Comment:
Maybe add a docstring to this theorem, explaining that this property follows from our choice of junk value for the adjoint operator of an operator which is not densely defined.